sdk/keyvault/azure-security-keyvault-jca/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 
 ## 1.0.0-beta.3 (2021-01-20)
+ - Fix NPE: https://github.com/Azure/azure-sdk-for-java/issues/17953
```

Comment:
I think we should give a clear outline of each change log instead of list links to github issues. Besides, please follow the specification of [release notes](https://azure.github.io/azure-sdk/policies_releasenotes.html#whats-a-developer-impacting-change) and add proper section headers.

Reply:
@yiliuTo , thank you for your suggestion. I updated it.